modin/_compat/pandas/py36/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+from .base import Python36CompatibleBasePandasDataset
+from .dataframe import Python36CompatibleDataFrame
+from .series import Python36CompatibilitySeries
+
+__all__ = [
+    "Python36CompatibleBasePandasDataset",
+    "Python36CompatibleDataFrame",
+    "Python36CompatibilitySeries",
```

Comment:
```suggestion
    "Python36CompatibleSeries",
```

Reply:
Compatibility -> Compatible, where applicable

Reply:
I would rather consult with native English speakers, it's unclear which form is better to me.

cc @modin-project/modin-contributors